transports/janus_mqtt.c

```diff
@@ -377,12 +377,12 @@ int janus_mqtt_init(janus_transport_callbacks *callback, const char *config_path
 
 		ctx->ssl_enabled = TRUE;
 
-		janus_config_item *cacertfile = janus_config_get(config, config_general, janus_config_type_item, "cacertfile");
+		janus_config_item *cacertfile = janus_config_get(config, config_general, janus_config_type_item, "cacertfile");		
```

Comment:
Looks like there's an extra tab at the end of this line that should be removed.

Reply:
Does that look better now?
Do I need to squash/compress the two commits somehow?

Reply:
No need, I can squash them when merging, thanks!

Reply:
Awesome! Thanks for being gentle about my little goofs.